onnx/defs/schema.cc

```diff
@@ -115,22 +114,22 @@ void OpSchema::Verify(const NodeProto& node) const {
     }
   }
 
-  for (size_t out_idx = 0; out_idx < static_cast<size_t>(node.output_size());
```

Comment:
Why not leave size_t as the type?

Reply:
Good point. You mean the return type of node.output_size(), right? will revisit the API later. This is fixing the warnings as I'm turning on taking warnings as errors. :)

Reply:
yes, unintended narrowing casts can be a source of problems. I would leave them as size_t if possible.